### ISSUE TITLE
Incorrect icons on certain themes after upgrading to Linux Mint v. 20.1 from Linux Mint v. 20.0 - Cinnamon

### ISSUE THREAD

**Issue by DorianB2017** [linuxmint/cinnamon#9886]
```
OS: Linux Mint 20.1 x86_64;
Cinnamon: v. 4.8.6;
Theme: Numix [GTK2/3];
Icons: Numix [GTK2/3].
 ```
After upgrading Linux Mint v. 20.0 - Cinnamon to Linux Mint ver. 20.1 - Cinnamon, for certain themes (icon themes), the icon for Nemo is only the one from Adwaita.
I use the Numix theme, with icon-numix-theme. The other icons in the theme have remained the same. (A similar issue was in the summer (June-July 2020) with the Discs icon in Settings when it was upgraded from Linux Mint v. 19.3 to Linux Mint ver. 20.0. The case was that the Linux Mint team forgot to change somewhere from "discs "in" gnome-discs ", the problem from then on was corrected only in September 2020. I think that here is the same problem with the name of the Nemo program, written somewhere wrong in the system.)

Before upgrading: "Captură de ecran din 2021-01-22 12-35-24_p.png".
![Captură de ecran din 2021-01-22 12-35-24_p](https://user-images.githubusercontent.com/30018486/105492065-07d7ee00-5cc0-11eb-91cd-9f8ae392e674.png)

After upgrading: "Captură de ecran din 2021-01-22 13-30-37_p.png".
![Captură de ecran din 2021-01-22 13-30-37_p](https://user-images.githubusercontent.com/30018486/105492127-1f16db80-5cc0-11eb-92e9-887661de24d2.png)

For the above, I bring proof that other Linux Mint users also encounter this problem with the Nemo icon in Linux Mint 20.1 after upgrade. Read here: https://forums.linuxmint.com/viewtopic.php?f=47&t=339742 .

Best regards.

**Comment by NikoKrause** [comment]
The icon name for nemo was changed to `system-file-manager`. Numix is probably missing this icon name. This needs to be added in numix. For the time being creating a symbolic link with this icon name in your theme manually is a workaround.

**Comment by DorianB2017** [comment]
> The icon name for nemo was changed to `system-file-manager`. Numix is probably missing this icon name. This needs to be added in numix. For the time being creating a symbolic link with this icon name in your theme manually is a workaround.

Please read carefully what I wrote in the above notification/issuse. The same problem was with the upgrade from Linux Mint ver. 19.3 to Linux Mint ver. 20.0, from the summer when the Linux Mint team changed the name "Discs" (just remember what the old technical name was) to the new name "gnome-disk ..." and the icon "Discs" was missing in "Settings". After 2-3 months they found the correspondence solution of the name "Discs".
And now you can do the same.
As I wrote in the above notification/issuse, other users, who have posted extra Linux Mint themes, have the same problem, even putting a proving link in this regard.

**Comment by NikoKrause** [comment]
> > The icon name for nemo was changed to `system-file-manager`. Numix is probably missing this icon name. This needs to be added in numix. For the time being creating a symbolic link with this icon name in your theme manually is a workaround.
> 
> Please read carefully what I wrote in the above notification/issuse. The same problem was with the upgrade from Linux Mint ver. 19.3 to Linux Mint ver. 20.0, from the summer when the Linux Mint team changed the name "Discs" (just remember what the old technical name was) to the new name "gnome-disk ..." and the icon "Discs" was missing in "Settings". After 2-3 months they found the correspondence solution of the name "Discs".
> And now you can do the same.
> As I wrote in the above notification/issuse, other users, who have posted extra Linux Mint themes, have the same problem, even putting a proving link in this regard.

Please read carefully what I wrote in the above answer. Can you confirm that your icon theme has an icon called `system-file-manager`?

**Comment by leigh123linux** [comment]
> > The icon name for nemo was changed to `system-file-manager`. Numix is probably missing this icon name. This needs to be 
> And now you can do the same.

File the issue against your busted theme!

**Comment by leigh123linux** [comment]
> > > 
> > > And now you can do the same.
> 
> File the issue against your busted theme!

**Comment by DorianB2017** [comment]
I apologize for not making myself understood. The same problem was with the "Discs" icon when Linux Mint was updated from version 19.3 to version 20.0 (June-July 2020), namely: the lack of the icon because the old name was called "disks" and had to be called "gnome-disks ". The correction was made by Clem only in September (2020) and after the correction, the icon appeared correctly.
I insist that it be corrected in the same way with the icon corresponding to Nemo, associating him with the new name.
Thank you for understanding.

**Comment by NikoKrause** [comment]
I apologize for not making myself understood.
I insist that you tell me if your icon theme has an icon called `system-file-manager`?

**Comment by DorianB2017** [comment]
Here: ![image](https://user-images.githubusercontent.com/30018486/106891582-8d619200-66f3-11eb-86b3-ec5ad006d2cf.png), the Nemo icon shows the correct one related to the Numix theme - icon-numix-theme.
So the problem is when upgrading to Linux Mint ver. 20.1.
And others are complaining about this ... [](https://forums.linuxmint.com/viewtopic.php?f=47&t=339742).

**Comment by NikoKrause** [comment]
That's not an answer to my question.

**Comment by DorianB2017** [comment]
There is no icon in the Numix theme series - icon-numix-theme - named `system-file-manager`, but only gnome-folder.svg (link) and folder.svg that were / are associated with Nemo.
![image](https://user-images.githubusercontent.com/30018486/106895235-a751a380-66f8-11eb-843b-c336ff815f21.png)

**Comment by NikoKrause** [comment]
Can you try and create a symbolic link called `system-file-manager.svg` which points to `gnome-folder. svg`? A reboot after creating the link may be needed.

**Comment by DorianB2017** [comment]
> Can you try and create a symbolic link called `system-file-manager.svg` which points to `gnome-folder. svg`? A reboot after creating the link may be needed.

It went with `system-file-manager.svg` as a link, but only after I executed the `gtk-update-icon-cache` command did the `system-file-manager.svg` icon appear (for Nemo). Until I executed the command, it didn't work with resetting the system, nor did it work with resetting the Cinnamon interface.
![image](https://user-images.githubusercontent.com/30018486/106900788-a07a5f00-66ff-11eb-9dc3-a1850ce45337.png)

But is it really not possible to associate Nemo with the new name of `system-File-manager` and not to have this problem with missing icons in various themes as was done in the case of" Discs "?

Anyway, thank you very much for help, NikoKrause.

Best regards.
